roles/parse_c2r_report/meta/main.yml

```
@@ -0,0 +1,52 @@
+galaxy_info:
+  author: your name
+  description: your role description
+  company: your company (optional)
+
+  # If the issue tracker for your role is not on github, uncomment the
+  # next line and provide a value
+  # issue_tracker_url: http://example.com/issue/tracker
+
+  # Choose a valid license ID from https://spdx.org - some suggested licenses:
+  # - BSD-3-Clause (default)
+  # - MIT
+  # - GPL-2.0-or-later
+  # - GPL-3.0-only
+  # - Apache-2.0
+  # - CC-BY-4.0
+  license: license (GPL-2.0-or-later, MIT, etc)
+
+  min_ansible_version: 2.1
```

We should match the version here: https://github.com/oamg/infra.convert2rhel/pull/3, which is 
```suggestion
  min_ansible_version: 2.14
```

---

if you merge mine in first this one will need to rebase anyways but that would probably be for the best

---

I updated also common role which was merged before, all roles now have `2.14`